### DISCUSSION TITLE
I Want I (Rather Than One)

### DISCUSSION THREAD

**Discussion by RieksJ** [AmpersandTarski/Ampersand#1247] [Ideas]
Being currently heavily involved in all sorts of (self-sovereign) identity stuff, it occurred to me that the populations of what we have come to call 'belly-context's are in fact a partial representation of the knowledge of what I have come to call a 'party', i.e. someone/something that is capable of perceiving the world, ordering its perceptions (e.g. in terms of concepts, relations and rules), reasoning with that and while doing so increasing in knowledge and experience - typical examples of 'parties' would be individual people and organizations (i.e. sets of people that sort of pursue some given set of goals).

I have been contemplating the meaning of parties being 'self-sovereign', and concluded that it means that each of them individually gets to decide what to perceive, how to order that, what kinds of reasoning is acceptable on that, etc.

Therefore, I would like to argue that the 'God-atom' in belly-contexts, which we currently denote as `"ONE"[ONE]`, would be much better expressed as `"I"[I]`, which would then mean that we use the phrase `I` as the identity-atom, as the identity concept/context, and as the identity relation. This appeals to me because I find it to be aesthetically nice.

So how about renaming the concept `ONE` and the atom `ONE` both into `I`?

**Comment by Michiel-s** [comment]
I find this very confusing. Especially because we also have the keyword I for the identity relation. Furthermore I think we shouldn’t try to mix self sovereign- and Ampersand terminology. A mapping on conceptual level is possible anyway.

**Comment by sjcjoosten** [comment]
Technical limitations to name clashes between concepts (like ONE) and expressions (like I) can be overcome (I believe some statements allow us to assign things to both concepts and relations, like POPULATE, I'm not too sure if there are potential clashes between concepts and expressions), but given that we find the difference between relations and concepts important enough to have a syntactical difference between them (starting with a capital or not), I think this is a bad idea in general.
In this specific case I think it's an even worse idea. However, you're already free to use I[ONE] and even V[ONE] in place of "ONE"[ONE]. In your proposal, I and "I" would be synonyms if they are both expressions, which seems utterly confusing, especially because they are of different types and therefore cannot be used interchangeably everywhere.

**Comment by RieksJ** [comment]
@Michiel-s: while I appreciate your confusion, and I agree we should try to reduce that to its bare minimum, for now I don't find it a convincing argument for not having `I` as a concept. The discussion I intend to do here is too fundamental for that.

@sjcjoosten: to me it is not 'given' that the importance of the difference between relations and concepts warrants syntactical differences. It has been decided a long time ago, and I imagine it just happened to make the parser of that time easier. I disagree with your suggestion that changing unchecked assumptions is a bad idea in general. 

I think we have long been in agreement that a 'God-concept' and a singular 'God-atom' which means that we can express 'God exists', and we can define and use relations using that concept and atom (which allows us e.g. to express e.g. 100 names of God, and reach any atom in the universe). However, we also have the ability to cater for the existence of different truths, i.e. distinct sets of concepts, relations and rules whose population is allowed to violate rules in other sets. We use the term 'context' to refer to such sets, and we have begun to think about the consequences that has for Ampersand - the 'multi-context' idea.

I think we need a concept and an atom that are 'an image of God', that allows us to express 'I exist', and that we can use to define and use relations with, so as e.g. to express (a) name(s) of 'I', and more. Let me use the term 'I-concept' and 'I-atom' for this. And since we use a different name, i.e. `ONE`, to refer to the God-concept and God-atom, I don't mind using a different name, e.g. `ME`, or `I-Am`, to refer to the I-concept and I-atom. The difference between the God-concept/atom and the `I-Am`-concept/atom is that the latter is context specific: every context has precisely one.

**Comment by hanjoosten** [comment]
@RieksJ , I think you missed the point that @sjcjoosten pointed out. I is not a concept. It just isn't. It is an expression.

**Comment by RieksJ** [comment]
@hanjoosten, `I` is neither an expression nor a concept. `I` is (currently) a text that is used to refer to an expression. It is a common practice that texts that refer to something are overloaded. For example, your kids would refer to you by `papa` (daddy), in the same way that my kids use that text to refer to me. As another example, FormalAmpersand uses `in` (as well as other texts) to refer to a variety of relations, using the signature mechanism to distinguish between them. So although I acknowledge that in the current setup `I` refers to an expression (or: relation), I fail to see why this must forever be the case, i.e. why it would be extensionally impossible for `I` to refer to something else than the expression it currently represents.

**Comment by Michiel-s** [comment]
As this is currently more a discussion than an issue/bug or concrete feature request I'll transfer this issue to the discussion section.
